### PR TITLE
Fixes mime type validation

### PR DESCRIPTION
**Pull request by djvirgen**
Fixes a bug where allowing multiple mime types can cause validation to fail on valid mime types.

### CODE REVIEW

File: lib/Request.js [hunk 1]
```diff
@@ -414,13 +419,13 @@ function validateMimeType(mimeType, allowedMimeTypes) {
         regex = regex.replace('::MATCHANY::', '[^/]+');
         regex = new RegExp('^' + regex + '$', 'i');
 
-        if (null === mimeType.match(regex)) {
-            return false;
+        if (null !== mimeType.match(regex)) {
```
Comment (mariane-bekker) [inline lib/Request.js:422]: can null ever happen? on line 407 we are changing null to ''

**Comment by kelfman** [comment]
:100:

File: lib/Request.js (hunk 1 above)
Comment (skylargivens) [inline lib/Request.js:422]: `.match` will return `null` if no matches are found.

**Comment by skylargivens** [comment]
:+1:

**Pull request merged**